azure-pipelines.yml

```diff
@@ -28,52 +28,52 @@ steps:
       npm run buildci
     displayName: build ci
 
-  - task: AzureUpload@1
-    displayName: Upload PR deploy site
-    inputs:
-      SourcePath: 'apps/pr-deploy-site/dist'
-      azureSubscription: 'UI Fabric (bac044cf-49e1-4843-8dda-1ce9662606c8)'
-      storage: fabricweb
-      ContainerName: '$web'
-      BlobPrefix: 'pr-deploy-site/$(Build.SourceBranch)'
+  # - task: AzureUpload@1
```

Comment:
This script will DEFINITELY be uncommented before check-in of whatever fix I find, it's just commented for now to speed things up...